documentation/modules/configuring/con-mm2-recovery.adoc

```diff
@@ -0,0 +1,44 @@
+// This module is included in:
+//
+// assembly-config.adoc
+
+[id="con-mm2-recovery-{context}"]
+= Disaster recovery in an active/passive configuration
+
+[role="_abstract"]
+MirrorMaker 2 can be configured for active/passive disaster recovery.  
+To support this, the Kafka cluster should also be monitored for health and performance to detect issues that detect issues that require failover promptly.
+
+In the event of failover, operations switch from the active cluster to the passive cluster when the active cluster becomes unavailable.
+
+In a worst-case scenario, the original active cluster is typically considered permanently lost.
+The passive cluster is promoted to active status, taking over as the source for all application traffic.
+In this state, MirrorMaker 2 stops replicating data to the original active cluster because it is no longer functional.
+
+Failback, or restoring operations to the original active cluster, requires careful planning.
+While it is technically possible to reverse roles by swapping the source and target clusters in the MirrorMaker 2 configuration, this approach carries the risk of data duplication when records mirrored to the passive cluster are mirrored back to the original active cluster. To avoid duplicates, resetting consumer offsets is an option, but it adds further complexity.
```

Comment:
I don't think you can really reverse the roles in MM2 configuration. You can in theory deploy a new MM2 for the other direction.

Reply:
I think we can change the wording in that first sentence: "While it is technically possible to reverse roles in MirrorMaker 2 by swapping the source and target clusters and deploying this configuration as a new instance..."

Reply:
Yeah, I think that would work.